src/WebConnector/Messages/StepQueryResponseBase.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace QbSync.WebConnector.Messages
```

Comment:
I think this file can be dropped. It was not in the solution before.

Reply:
Aha!  That's probably an artifact of shifting to the new csproj format which uses globbing to find all the files in the project instead of explicitly including them.  I will fix this.